README.md

```diff
@@ -44,7 +54,7 @@ A quick demo of how to submit a calculation:
 ```shell
 verdi daemon start     # make sure the daemon is running
 cd examples
-verdi run submit_singlepoint.py "janus@localhost" --calctype "singlepoint"  --architecture mace --model "~./cache/mlips/mace/46jrkm3v"       # run test calculation
+verdi run submit_singlepoint.py "janus@localhost" --architecture mace --model "~./cache/mlips/mace/46jrkm3v"    # run test calculation
```

Comment:
```suggestion
verdi run submit_singlepoint.py "janus@localhost" --architecture mace --model "~/.cache/mlips/mace/46jrkm3v"    # run test calculation
```
Probably?

Alternatively, we may want a more generic /path/to/model, especially since this doesn't come with the git repo?

Reply:
yeah I think more generic makes more sense